nemo/collections/asr/modules/rnnt.py

```diff
@@ -50,6 +50,525 @@
 from nemo.utils import logging
 
 
+class StatelessNet(torch.nn.Module):
+    def __init__(self, context_size, vocab_size, emb_dim, blank_idx, blank_as_pad, normalization_mode, dropout):
+        super().__init__()
+        self.context_size = context_size
+        self.vocab_size = vocab_size
+        self.emb_dim = emb_dim
+        self.dropout = torch.nn.Dropout(dropout)
+        self.norm = torch.nn.Identity()
+        if normalization_mode == 'layer':
+            self.norm = torch.nn.LayerNorm(emb_dim, elementwise_affine=False)
+
+        assert(blank_as_pad)  # So far it only supports when it is set True
```

Comment:
Its not necessary to have this then, it was mainly because we were overloading the Embedding Blank as zero vector for more efficient decoding. Newer decoders can just default to this.

Reply:
Done.